artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireConnection.java

```diff
@@ -286,6 +295,9 @@ public void bufferReceived(Object connectionID, ActiveMQBuffer buffer) {
                dispatchSync(response);
             }
          }
+         else {
+            System.out.println(this + "====server received a keepalive: " + System.currentTimeMillis());
```

Comment:
System.out??????   use a Logger.getLogger(). then trace.. look at how trace is done in other places.